.gitignore

```diff
@@ -37,9 +37,6 @@ __pycache__/
 .stylelintcache
 *.tsbuildinfo
 
-# T1 emulation file
-docker/trezor-user-env/controller/emulator.img
```

Comment:
I believe it's a leftover from https://github.com/trezor/trezor-suite/commit/879220c39858d2051728b511451915a3555dd5c0